src/Sparse/Wiedemann.jl

```diff
@@ -0,0 +1,164 @@
+@doc Markdown.doc"""
```

Comment:
I'd also add a top-level function like `some_nullspace` taking a SMat(A) and calling `wiedemann(A, transpose(A))`

Reply:
Like this?

Reply:
@fieker like this?